### PR TITLE
support complex polygons

### PR DESCRIPTION
**Pull request by yeesian**
Please review before merging

### CODE REVIEW

File: src/patchwork.jl [hunk 1]
```diff
@@ -219,13 +219,17 @@ function draw(img::Patchable, prim::PolygonPrimitive)
     paths = make_paths(prim.points)
     if length(paths) > 1
         addto(Elem(:svg, :g)
-              [Elem(:svg, :path, fill="none", d=svg_fmt_path(path, true) * " z")
```
Comment (dcjones) [inline src/patchwork.jl]: @shashi Did we determine that these `fill` attributes were superfluous?

Comment (yeesian) [inline src/patchwork.jl]: I think he dealt with them in https://github.com/dcjones/Compose.jl/commit/6172c19df0a016b9a20157ff7d72ac8322623219.

Comment (dcjones) [inline src/patchwork.jl]: Oh, ok!

**Comment by dcjones** [comment]
Rebased and merged. Thanks! :+1:

**Pull request closed without merging**